macros/edr/system/system_utils/buckets_cte.sql

```diff
@@ -0,0 +1,58 @@
+{% macro complete_buckets_cte(time_bucket) %}
+    {{ adapter.dispatch('complete_buckets_cte','elementary')(time_bucket) }}
+{% endmacro %}
```

Comment:
Just adding a note to make sure these methods are well tested on all platforms including Databricks